File: scholar-business/src/main/java/com/varun/scholar/business/data/entities1/AttendanceSearch.java
package com.varun.scholar.business.data.entities1;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class AttendanceSearch extends AbstractEntity {
	String employeeId;
	String staffName;
	String department;
	@Temporal(TemporalType.DATE)
	Date lastChange;
}

File: scholar-web/src/main/webapp/app/model/staff/attendance/AttendanceSearch.js
Ext.define('scholar.model.staff.attendance.AttendanceSearch', {
	extend: 'Ext.data.Model',
	fields : [ {
		name : 'id',
		type : 'int'
	}, {
		name : 'employeeId',
		type : 'string'
	}, {
		name : 'staffName',
		type : 'string'
	}, {
		name : 'department',
		type : 'string'
	},  {
		name : 'lastChange',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ],
	
});